stix_shifter/stix_translation/src/modules/guardium/data_mapping.py

```diff
@@ -0,0 +1,36 @@
+from os import path
```

Comment:
You shouldn't need all of this in your data mapping since it's already included in the `BaseDataMapper` which your `DataMapper` class inherits. If you just copy the dummy translator's `data_mapping.py` it should work.

Reply:
Please review.